modules/virt-about-runstrategies-vms.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-create-vms.adoc
+
+[id="virt-about-runstrategies-vms_{context}"]
+= About RunStrategies for virtual machines
+
+Virtual machines have a `RunStrategy` setting to determine whether or not a guest should be running a virtual machine instance.
```

Comment:
I think that this needs to be expanded upon. As I understand it we set the RunStrategy for a VM to determine whether or not a VMI will be created/destroyed based on a set of circumstances. Those circumstances are captured by the particular RunStrategy of the VM.
We should also cover how this is different to the previous method, which is `spec.running: true`, since these are mutually exclusive and users will need to know when they should use one or the other.